proteus/mprans/RANS3PF2D.h

```diff
@@ -5505,9 +5506,7 @@ namespace proteus
                                      vf[eN_k],
                                      phi[eN_k],
                                      &normal_phi[eN_k_nSpace],
-                                     nParticles,
-                                     nQuadraturePoints_global,
-                                     &particle_signed_distances[eN_k],
+                                     phi_solid[eN_k],//computed in getResidual
```

Comment:
Here, I would be consistent with using `distance_to_omega_solid` as the input variable to evaluateCoefficients.